btr-api/src/btr_api/config.py

```diff
@@ -125,6 +125,13 @@ class Config:  # pylint: disable=too-few-public-methods
 
     LEGISLATIVE_TIMEZONE = os.getenv('LEGISLATIVE_TIMEZONE', 'America/Vancouver')
 
+    # Cache stuff
+    CACHE_TYPE = os.getenv('CACHE_TYPE', 'SimpleCache')
+    try:
+        CACHE_DEFAULT_TIMEOUT = int(os.getenv('CACHE_DEFAULT_TIMEOUT', '300'))
+    except (TypeError, ValueError):
```

Comment:
At this point logging won't be set up so it would have to be a print statement. I don't think it matters. We don't do this anywhere else with similar sort of things. It's generally just there to set a default when nothing is provided in the .env